charts/lakefs/templates/deployment.yaml

```
@@ -19,6 +19,16 @@ spec:
       labels:
         {{- include "lakefs.selectorLabels" . | nindent 8 }}
     spec:
+      {{- if eq ( include "lakefs.fullname" .) "treeverse/lakefs-enterprise" }}
+       {{- if (.Values.fluffy.image.privateRegistry).enabled }}
+      imagePullSecrets:
+        {{- if (.Values.fluffy.image.privateRegistry).secretToken }}
+        - name: "docker-registry"
+        {{- else }}
+        - name: {{ .Values.fluffy.image.privateRegistry.secretName }}
+        {{- end }}
+       {{- end }}
+      {{- end }}
```

It doesn't really render, even when fixing ` include "lakefs.repository"`. 
How I checked: 
When running `helm template -f <some-enterprise-values.yaml> lakefs ./charts/lakefs > out.yaml` the `imagePullSecrets` is not being rendered at all. 
I thought maybe it related to the issue with how you check `include "lakefs.fullname"` but it's something else, here's a working example:

```suggestion
      {{- if and (.Values.fluffy).enabled .Values.fluffy.image.privateRegistry.enabled }}
      imagePullSecrets:
        {{- if .Values.fluffy.image.privateRegistry.secretToken }}
        - name: "docker-registry"
        {{- else }}
        - name: {{ .Values.fluffy.image.privateRegistry.secretName }}
        {{- end }}
        {{- end }}
```

If you run helm template on this you'll see the expected behavior. 
Notice in this suggestion I did not include `lakefs.repository` check, commented on it separately.